accord-core/src/main/java/accord/impl/InMemoryCommandStore.java

```diff
@@ -398,6 +397,14 @@ protected <T> void executeInContext(InMemoryCommandStore commandStore, PreLoadCo
         }
     }
 
+    @Override
+    public String toString()
```

Comment:
when you run in the wrong store we log the stores involved, so need this.  I felt that only `id` is useful so left everything else out.